docs/source/app-configuration.rst

```diff
@@ -0,0 +1,160 @@
+App Integration Configuration
+=============================
+
+Overview
+--------
+
+To further StreamAlert's abilities past ingesting logs from services that offer support for sending to AWS Lambda,
```

Comment:
Suggestion:

"""
For StreamAlert and other related platforms, log forwarding is usually left as an exercise to the reader. This work is non-trivial and often requires new infrastructure and code. We wanted to make this easier for everyone and have achieved this through StreamAlert Apps. 

StreamAlert Apps allow you to collect logs from popular services and applications in minutes. You simply provide the application's credentials and StreamAlert will deploy an individual serverless application that will fetch and forward logs to StreamAlert for analysis and alerting.
"""